deps/hashicorp/docker-compose.yml

```diff
@@ -1,126 +1,150 @@
 version: '3.7'
 
 x-container-common: &container-common
+  image: consensys/quorum-hashicorp-vault-plugin:v1.1.4
   restart: ${CONTAINER_RESTART-on-failure}
   networks:
     - hashicorp
-
+      
+x-default-variables: &default-variables
+  ROOT_TOKEN_PATH: /vault/token/.root
+  PLUGIN_MOUNT_PATH: quorum
+  KVV2_MOUNT_PATH: secret
+  VAULT_IS_READY: /vault/token/.ready
+  
+x-tls-variables: &tls-variables
```

Comment:
I'm not a fan of these defaults defined in docker-compose because most of the time the different services do not share the same ENV vars so we end up defining ENV vars for services that do not use them. I prefer defining the used ENV vars specifically per service even if there is a bit of copy-paste. 
Currently the agent does not use `PLUGIN_MOUNT_PATH` for example, yet it is defined in the service.

Reply:
Actually `PLUGIN_MOUNT_PATH` is used in the agent as well do define the policy capabilities. 

IMO this level of encapsulation fits quiet well into the needs we have here, because:
1. `default-vars` are used by every container
2. `tls-vars` by both tls container (agent and vault)
3. `agent-vars`by both agent containers

Reply:
if every var is indeed used by the services where they are defined then it's fine.